---
permalink: /
title: "Home"
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---
Welcome to my homepage!

I am currently a Ph.D. candidate in [Department of Biostatistics](https://sph.unc.edu/bios/biostatistics/) at the [University of North Carolina at Chapel Hill (UNC-CH)](https://www.unc.edu), 
advised by [Dr. Yun Li](https://sph.unc.edu/adv_profile/yun-li-phd/).
At UNC-CH, I am also working closely with [Dr. Karen Mohlke](https://www.med.unc.edu/genetics/directory/karen-l-mohlke-phd/), 
[Dr. Nora Franceschini](https://sph.unc.edu/adv_profile/nora-franceschini-md/),
[Dr. Christopher Sayed](https://www.med.unc.edu/derm/directory/christopher-sayed-md/)
and [Dr. Laura Raffield](https://www.med.unc.edu/genetics/directory/laura-raffield-phd/). 

Previously, I received my B.S. in Mathematics and Applied Mathematics from the [School of Mathematical Sciences](http://www.math.zju.edu.cn/mathen/) at [Zhejiang University](https://www.zju.edu.cn/english/).


Research Briefings
======
My research has been focusing on statistical genetics, including both developing statistical methods applicable to genetics fields, and analyzing genetics data to disentangle complex diseases and traits,
including a special focus on promoting diversity in genetics studies. 
Specifically, I have developed statistical methods for improving genotype imputation quality estimation ([MagicalRsq](https://pubmed.ncbi.nlm.nih.gov/36198314/)), 
performing genetics association testing ([eSCAN](https://pubmed.ncbi.nlm.nih.gov/34882196/)) and 
improving polygenic risk score (PRS) prediction accuracy in under-represented minority populations ([GAUDI](https://www.biorxiv.org/content/10.1101/2022.10.06.511219.abstract)). 
I have developed computational tools, including a linkage disequilibrium (LD) exploration tool ([TOP-LD](http://topld.genetics.unc.edu)) using 
whole genome sequencing (WGS) data from the [Trans-Omics for Precision Medicine (TOPMed) Project](https://topmed.nhlbi.nih.gov), 
and a functional annotation tool for hematological traits ([VAMPIRE](https://pubmed.ncbi.nlm.nih.gov/35047852/)) that tackles the variant-to-function problem. 
I have also been participating in genotype imputation and large-scale genome-wide association studies (GWAS), 
and have identified several novel signals that may guide future functional genomics studies and eventually impact clinical practices.
More recently, I am working on single-cell RNA sequencing (scRNA-seq) data, cell type deconvolution and cell-type-specific analysis.
In addition, I am also a member of several multi-institution consortium efforts, including the [TOPMed project](https://topmed.nhlbi.nih.gov), 
[Cystic Fibrosis Genome Project (CFGP)](https://www.cff.org/researchers/whole-genome-sequencing-project-data-requests), 
[Polygenic Risk Methods in Diverse Populations (PRIMED) consortium](https://primedconsortium.org),
[Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) consortium](https://www.chargeconsortium.com) 
and [Impact of Genomic Variation on Function (IGVF) consortium](http://igvf.org). 

**Key words**

  * Statistical genetics and genomics
  * Bioinformatics
  * Polygenic risk score
  * Diversity
  * Single-cell RNA-seq
  * High-dimensional data
  * Machine learning





